src/input-select/input-select.tsx

```diff
@@ -69,6 +70,9 @@ export const InputSelect = <T, V>({
         }
         setShowOptions(!showOptions);
         triggerOptionDisplayCallback(!showOptions);
+        if (showOptions) {
+            onBlur(event);
+        }
```

Comment:
onBlur is optional so optional chaining can be used here

also suggest applying the selector blur on the button instead of here so that it matches the visual focus

```
            <Selector
                onBlur={(e) => {
                    if (!showOptions) {
                        onBlur?.(e);
                    }
                }}
            >
```

Reply:
Actually this wont work because as mentioned `Selector` is a button and it's `onBlur` is the moment you release your click.

Reply:
what behaviour did you observe with this handler logic?

was thinking it could cover the scenario where user tabs between form inputs